src/Nethermind/Nethermind.Consensus/Producers/DevBlockProducer.cs

```diff
@@ -7,6 +7,7 @@
 using Nethermind.Consensus.Transactions;
 using Nethermind.Core;
 using Nethermind.Core.Specs;
+using Nethermind.Evm;
```

Comment:
Whats with these new `using`(s)?

Reply:
Had to move IBlocksConfig to EVM project

Reply:
made more sense to put it inside the config project since it is not related to the EVM so i moved it there.